petrinet/examples/halfar.json

```diff
@@ -0,0 +1,258 @@
+{
+  "header": {
+    "name": "Halfar Model",
+    "schema": "https://raw.githubusercontent.com/DARPA-ASKEM/Model-Representations/petrinet_v0.1/petrinet/petrinet_schema.json",
```

Comment:
Should this be referring to the latest schema version: `[petrinet_v0.6](https://github.com/DARPA-ASKEM/Model-Representations/releases/tag/petrinet_v0.6)`?

Reply:
Yes, will fix.